randobot/zsr.py

```diff
@@ -172,4 +174,15 @@ def load_available_settings(self):
         Settings pool for Draft Mode.
         """
         pool = requests.get(self.draft_settings_pool_endpoint).json()
-        return pool
\ No newline at end of file
+        return pool
+    
+    def handle_conditionals(self, dev, settings):
+        if dev:
+            preset = self.presets_dev.get('s7').get('settings')
+        else:
+            preset = self.presets.get('s7').get('settings')
+
+        for key, value in settings.items():
+            if key == 'shuffle_dungeon_entrances' and value == 'simple':
+                preset['allowed_tricks'].append('logic_dc_scarecrow_gs')
```

Comment:
Same issue here, in this case I think you can skip using `presets_dev` entirely and only use the data in `settings`

Reply:
I think I have to use the dev settings because 's7' doesn't exist otherwise. The plan is to make this only compatible with release once 8.0 is here which in that case everything will be pulled from presets instead of presets_dev